__tests__/app.test.js

```diff
@@ -165,4 +165,35 @@ describe('/api/articles/:article_id/comments', () => {
             expect(message).toBe('Bad Request')
         })
     })
+})
+
+describe('/api/articles/:article_id/comments', () => {
+    test('POST 201: Should return the posted comment', () => {
+        const newComment = {
+            body: 'I love pugs',
+            username: 'lurker'
+        };
+        return request(app)
+        .post('/api/articles/3/comments')
+        .send(newComment)
+        .expect(201)
+        .then(({ body: { comment }})=> {
+            expect(comment.author).toBe("lurker")
+            expect(comment.body).toBe("I love pugs")
+            expect(comment.votes).toBe(0)
+            expect(comment.article_id).toBe(3)
+        })
```

Comment:
What happens if there are additional properties that are passed? For example:
**const newComment = {
            body: 'I love pugs',
            username: 'lurker',
            newProperty: 'newProp'
        }**